packages/arb-token-bridge-ui/src/util/deposits/fetchDeposits.ts

```diff
@@ -49,23 +49,6 @@ export const fetchDeposits = async ({
     fromBlock = 0
   }
 
-  if (!toBlock) {
```

Comment:
For deposits, all tx history data comes from the subgraph (nothing from onchain), so it was unnecessary to fetch the latest block number, then set that to be `toBlock`, as it defaults to fetch up to latest

It's different for withdrawals where we actually use the `toBlock` to fetch everything from `[toBlock, latest]` from chain